### PR TITLE
Pass payload through to `receive`

### PR DESCRIPTION
**Pull request by BlakeWilliams**
When a `receive` callback is called pass back the received messages
`payload` to the callback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Fauxnix.inject(socket);
 
 Whenever `socket` receives a message from `phoenix.js` with a topic of
 `"topic"` and an event of `"event"` Fauxnix will send a reply with the given
-`status` and `response` in the return statement.
+`status` and `response` in the return statement. The `paylaod` from a message is
```
Comment (jakecraige) [inline README.md]: s/paylaod/payload

Comment (BlakeWilliams) [inline README.md]: Oops! Nice catch!

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ returned object of `new WebSocket()` calls with the socket you passed in.
 
 ```javascript
 const socket = new Fauxnix(function() {
-  this.receive("topic", "event", function() {
+  this.receive("topic", "event", function(payload) {
     return { status: "ok", response: { id: 1 } };
```
Comment (jakecraige) [inline README.md]: Should you use the `payload` in what you return here?

Comment (BlakeWilliams) [inline README.md]: It's more of an example than anything.

Would `response: { id: payload.id }` work?

Comment (jakecraige) [inline README.md]: Yeah that would be fine. I just wouldn't want it to confuse someone by there being this option and not being used or knowing what it is.

**Comment by jakecraige** [comment]
That 1 comment then LGTM

**Comment by BlakeWilliams** [comment]
Thanks for the review @jakecraige!

**Pull request merged**